terminal-scm-1.rockspec

```diff
@@ -40,6 +40,8 @@ build = {
     ["terminal.output"] = "src/terminal/output.lua",
     ["terminal.clear"] = "src/terminal/clear.lua",
     ["terminal.scroll"] = "src/terminal/scroll.lua",
+    ["terminal.color"] = "src/terminal/color.lua",
+    ["terminal.color.stack"] = "src/terminal/stack/color.lua"
```

Comment:
nitpick: 
```suggestion
    ["terminal.color.stack"] = "src/terminal/stack/color.lua",
```

if you add a trailing comma (allowed in Lua) then adding a new entry later will result in a diff of only 1 line, without the comma now, it will be 2 lines later.